presentation-compiler/test/dotty/tools/pc/tests/completion/BlaSuite.scala

```diff
@@ -0,0 +1,24 @@
+package dotty.tools.pc.tests.completion
+
+import dotty.tools.pc.base.BaseCompletionSuite
+import org.junit.Test
+
+class BlaSuite extends BaseCompletionSuite:
+  @Test
+  def t1 = check(
+    """
+    |object M:
+    | val VersionRegex = "".r
+    | VersionRe@@"1234"
+    """.stripMargin,
+    "|VersionRegex: Regex".stripMargin 
+  )
+
+  @Test
+  def t2 = check(
```

Comment:
Might also be worth to do a `checkEdit` test to see if we don't add () without needing to